server/service/service.go

```diff
@@ -52,49 +45,28 @@ func NewService(ds kolide.Datastore, resultStore kolide.QueryResultStore,
 	return svc, nil
 }
 
-// osqueryLogFile creates a log file for osquery status/result logs
-// the logFile can be rotated by sending a `SIGHUP` signal to kolide if
-// enableRotation is true
-func osqueryLogFile(path string, appLogger kitlog.Logger, enableRotation bool) (io.Writer, error) {
-	if enableRotation {
-		osquerydLogger := &lumberjack.Logger{
-			Filename:   path,
-			MaxSize:    500, // megabytes
-			MaxBackups: 3,
-			MaxAge:     28, //days
-		}
-		appLogger = kitlog.With(appLogger, "component", "osqueryd-logger")
-		sig := make(chan os.Signal)
-		signal.Notify(sig, syscall.SIGHUP)
-		go func() {
-			for {
-				<-sig //block on signal
-				if err := osquerydLogger.Rotate(); err != nil {
-					appLogger.Log("err", err)
-				}
-			}
-		}()
-		return osquerydLogger, nil
-	}
-	// no log rotation
-	return logwriter.New(path)
-}
-
 type service struct {
 	ds          kolide.Datastore
 	resultStore kolide.QueryResultStore
 	logger      kitlog.Logger
 	config      config.KolideConfig
 	clock       clock.Clock
 
-	osqueryStatusLogWriter io.Writer
-	osqueryResultLogWriter io.Writer
-
+	resultsQueues   []kolide.QueueService
+	statusQueues    []kolide.QueueService
 	mailService     kolide.MailService
 	ssoSessionStore sso.SessionStore
 	metaDataClient  *http.Client
 }
 
+func (s service) HealthCheckers() (map[string]interface{}) {
```

Comment:
To be honest this needs to be deleted and not done here.  I will fix that by moving this back to when the connections to services are created.